doc/Caliptra.md

```diff
@@ -894,29 +894,31 @@ Caliptra contains 32 PCR 384 banks that are extendable by the SHA engine, and re
 
 ### Attestation of Caliptra's update journey
 
-The Caliptra hardware ROM will extend Caliptra's FMC measurement and security state into the cumulative and current PCR banks (0 and 1 respectively).  The FMC captures the current measurement of Caliptra's Runtime Firmware upon every cold boot and hitless update. These measurements are used to derive a CDI and an alias key given to runtime firmware. FMC places runtime firmware's measurements into runtime firmware's alias key certificate, and signs that certificate with FMC's alias key.
+The Caliptra hardware ROM extends Caliptra's FMC measurement and security state into the cumulative and current PCR banks (0 and 1 respectively). The FMC captures the current measurement of Caliptra's runtime firmware upon every cold boot and hitless update. These measurements are used to derive a CDI and an alias key given to runtime firmware. FMC places runtime firmware's measurements into runtime firmware's alias key certificate, and signs that certificate with the FMC alias key.
```

Comment:
s/cumulative and current/current and cumulative/

(Just to make the following "(0 and 1 respectively)" accurate)

Reply:
should be fixed with merge conflict resolution but please open issue if needed.